jsonschema-validation.xml

```diff
@@ -514,6 +532,20 @@
                     format attribute and instance SHOULD succeed.
                 </t>
 
+                <t>
+                    Meta-schemas that do not use "$vocabulary" SHOULD be considered to
+                    require this vocabulary as if its URI were present with a value of true,
+                    although see the Implementation Requirements below for details.
```

Comment:
*also* see

(another occurrence below)

Reply:
@dlax I did actually mean "although" here, because the Implementation Requirements explains the weird optional nature of `format`, which is an exception to how the other vocabularies (except `content*`, which is the other "although") behave.

So I see this as a contrast to be called out ("although") rather than an affirmation or extension of the normal situation ("also").  Does that make sense?

Reply:
@handrews, yes, it makes sense. Thanks for clarifying!

Reply:
@dlax OK to resolve this?

Reply:
@Relequestual sure! It was already "resolved" from my point of view :)